pom.xml

```diff
@@ -7,7 +7,7 @@
     <groupId>au.org.sports.wrugby</groupId>
     <artifactId>scoreboard</artifactId>
     <version>1.0-SNAPSHOT</version>
-    <packaging>war</packaging>
+    <packaging>jar</packaging>
```

Comment:
Need to revert this change.  war packaging is correct for spring-boot and then run with java -jar scoreboard-1.0-SNAPSHOT.war.  The war file contains the embedded tomcat.